test/tests/functional/pbs_hook_set_jobenv.py

```diff
@@ -127,6 +128,8 @@ def tearDown(self):
             os.remove(self.job_out1_tempfile)
             os.remove(self.job_out2_tempfile)
             os.remove(self.job_out3_tempfile)
+            self.du.rm(self.mom.hostname, self.job_out1_tempfile)
+            self.du.rm(self.mom.hostname, self.job_out2_tempfile)
```

Comment:
Notice ,Line 131 and 132 is throwing error in tearDown as below in test output logs:-
2020-08-14 03:13:47,142 INFOCLI2 r1i0n0: ssh r1i0n0.ib0.smc-default.chf.rdlabs.hpecorp.net /usr/bin/rm /tmp/job_out1rml3t0l8
2020-08-14 03:13:47,423 ERROR    err: ["/usr/bin/rm: cannot remove '/tmp/job_out1rml3t0l8': No such file or directory"]
2020-08-14 03:13:47,424 INFOCLI2 r1i0n0: ssh r1i0n0.ib0.smc-default.chf.rdlabs.hpecorp.net /usr/bin/rm /tmp/job_out2iv75u2ft
2020-08-14 03:13:47,701 ERROR    err: ["/usr/bin/rm: cannot remove '/tmp/job_out2iv75u2ft': No such file or directory"]

Think this is because we already remove file in line 128 and 129.

Reply:
Hi Lovely, this error is ignorable as the file is not there, there is no harm to run an rm command on that.